packages/types-algosdk/types/index.d.ts

```diff
@@ -195,10 +195,25 @@ declare module 'algosdk' {
     v: number;
   }
 
-  export interface MultiSigAccount {
+  /**
+   * Src: https://github.com/algorand/js-algorand-sdk/blob/develop/src/types/multisig.ts#L6
+   */
+  // TODO: use latest typings from SDK
+  export interface MultisigMetadata {
```

Comment:
There is a very similiar interface `MultiSigAccount`. 

Also, if we absolutely need this interface then maybe we can change this to `MultiSigParams` and also use this at lines 362, 469, etc.

Reply:
> There is a very similiar interface MultiSigAccount.

Replaced this because SDK uses `MultiSigAccount`. We'll soon be migrating to ts-sdk.